src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -236,69 +367,56 @@ open class HiddenSecretsPlugin : Plugin<Project> {
          */
         project.task(TASK_HIDE_SECRET)
         {
+            this.group = TASK_GROUP
+            this.description = "Obfuscate a key and add it to your Android project"
             dependsOn(TASK_UNZIP_HIDDEN_SECRETS)
 
             doLast {
                 //Assert that the key is present
                 getKeyParam()
                 //Copy files if they don't exist
-                copyCppFiles()
-                copyKotlinFile()
+                copyCppFiles(false)
+                copyKotlinFile(false)
 
                 val keyName = getKeyNameParam()
                 val packageName = getPackageNameParam()
                 val obfuscatedKey = getObfuscatedKey()
 
-                //Add method in Kotlin code
-                var secretsKotlin = getKotlinFile()
-                if (secretsKotlin == null) {
-                    //File not found in project
-                    secretsKotlin = getKotlinDestination(packageName, KOTLIN_FILE_NAME)
-                }
-                if (secretsKotlin.exists()) {
-                    var text = secretsKotlin.readText(Charset.defaultCharset())
-                    text = text.replace(PACKAGE_PLACEHOLDER, packageName)
-                    if (text.contains(keyName)) {
-                        println("⚠️ Method already added in Kotlin !")
-                    }
-                    text = text.dropLast(1)
-                    text += CodeGenerator.getKotlinCode(keyName)
-                    secretsKotlin.writeText(text)
-                } else {
-                    error("Missing Kotlin file, please run gradle task : $TASK_COPY_KOTLIN")
-                }
-                //Resolve package name for C++ from the one used in Kotlin file
-                var kotlinPackage = Utils.getKotlinFilePackage(secretsKotlin)
-                if (kotlinPackage.isEmpty()) {
-                    println("Empty package in $KOTLIN_FILE_NAME")
-                    kotlinPackage = packageName
-                }
+                hideSecret(
+                    keyName = keyName,
```

Comment:
Not need to repeat names and vars: 
`hideSecret(keyName, packageName, obfuscatedKey)`